packages/mui-codemod/src/deprecations/avatar-props/avatar-props.js

```diff
@@ -0,0 +1,56 @@
+import findComponentJSX from '../../util/findComponentJSX';
+import assignObject from '../../util/assignObject';
+import appendAttribute from '../../util/appendAttribute';
+
+/**
+ * @param {import('jscodeshift').FileInfo} file
+ * @param {import('jscodeshift').API} api
+ */
+export default function transformer(file, api, options) {
+  const j = api.jscodeshift;
+  const root = j(file.source);
+  const printOptions = options.printOptions;
+
+  findComponentJSX(j, { root, componentName: 'Avatar' }, (elementPath) => {
```

Comment:
We should look into an abstraction for the slot pattern standardization codemods, as all will be pretty similar. It doesn't need to happen now, we can wait until we have a couple more.